### PR TITLE
ap blur focus update

### PR DESCRIPTION
**Pull request by itanandp**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -183,6 +183,14 @@ export default class RNPickerSelect extends PureComponent {
     }
 
     togglePicker(animate = false) {
+        if (this.state.showPicker) {
+            this.props.onBlur()
+        }
+        else {
```
Comment (gsteigerwald) [inline src/index.js]: `} else {` :trollface:

Comment (itanandp) [reply src/index.js]: @gsteigerwald you out here reviewing faster than my actual team members 😂

File: src/index.js [hunk 2]
```diff
@@ -183,6 +183,13 @@ export default class RNPickerSelect extends PureComponent {
     }
 
     togglePicker(animate = false) {
+        if (this.state.showPicker) {
+            this.props.onBlur()
+        } else {
+            this.props.onFocus()
+        }
+
```
Comment (gsteigerwald) [inline src/index.js:191]: extra white space :trollface:

Comment (ph-flip) [reply src/index.js:191]: this guy

Comment (itanandp) [reply src/index.js:191]: making me look baaaaaaad. But good catch thank you ;)

Comment (gsteigerwald) [reply src/index.js:191]: 👍 🥇

**Pull request merged**